frontend/taipy-gui/src/components/Taipy/Metric.tsx

```diff
@@ -144,34 +145,40 @@ const Metric = (props: MetricProps) => {
             layout.template = template;
         }
 
-        return layout
+        return {
```

Comment:
no need to expand layout here
I propose
```
if (props.title) {
   layout.title = props.title;
}
```

Reply:
resolved